packages/portfolio-contract/test/portfolio.contract.test.ts

```diff
@@ -355,3 +355,96 @@ test('open portfolio with USDN, Aave positions', async t => {
   t.snapshot(contents, 'vstorage');
   t.snapshot(done.payouts, 'refund payouts');
 });
+
+test.only('open portfolio with USDN position and withdraw funds', async t => {
+  const { trader1, common } = await setupTrader(t);
+  const { usdc, poc24 } = common.brands;
+
+  // First, open a portfolio with USDN position
+  const doneP = trader1.openPortfolio(
+    t,
+    {
+      USDN: usdc.units(3_333),
+      NobleFees: usdc.make(100n),
+      Access: poc24.make(1n),
+    },
+    { destinationEVMChain: 'Ethereum' },
+  );
+
+  // ack IBC transfer for forward
+  await common.utils.transmitVTransferEvent('acknowledgementPacket', -1);
+
+  const done = await doneP;
+  const result = done.result as any;
+  t.is(passStyleOf(result.invitationMakers), 'remotable');
+  t.like(result.publicSubscribers, {
+    portfolio: {
+      description: 'Portfolio',
+      storagePath: 'orchtest.portfolios.portfolio0',
+    },
+  });
+
+  // Verify portfolio was created successfully
+  const { storagePath } = result.publicSubscribers.portfolio;
+  t.log('Portfolio created at:', storagePath);
+  let portfolioInfo = getPortfolioInfo(storagePath, common.bootstrap.storage);
+
+  // Verify the initial USDN position
+  const { positionPaths } = portfolioInfo;
+  t.is(
+    portfolioInfo.contents[positionPaths[0]].accountId,
+    `cosmos:noble-1:cosmos1test`,
+  );
+
+  // Now proceed with withdrawal
+  t.log('Beginning withdrawal from USDN position');
+
+  // Request withdrawal of 1000 USDC from USDN position
+  const withdrawalAmount = usdc.units(3_333);
```

Comment:
```suggestion
  // Request withdrawal of 1000 USDC from USDN position
  const withdrawalAmount = usdc.units(1_000);
```

or eliminate the possibility of these getting out of sync:

```suggestion
  // Request withdrawal from USDN position
  const withdrawalAmount = usdc.units(3_333);
```